test/tests/ver-1g/comparison_ver-1g.py

```diff
@@ -12,25 +12,23 @@
 
 # ==============================================================================
 
-analytical_time = np.linspace(0.0, 40, 40)
+tmap_sol = pd.read_csv("./gold/ver-1g_diff_conc_out.csv")
+tmap_time = tmap_sol['time']
+tmap_concAB = tmap_sol['conc_ab']
+ax.plot(tmap_time, tmap_concAB, label=r"TMAP8", c='tab:gray')
+
+analytical_time = np.linspace(0.0, 40, 100)
 concA_o = 2.43e-4
-concB_o = 1.21501e-4  # Added 0.00001e-4 to actual value to avoid divide-by-zero
-# error in the analytical solution
-concAB_o = 0
+concB_o = 1.215e-4
```

Comment:
We need to be very clear about what we are doing here:
- In TMAP4, the partial pressures are 1e-6 Pa and **0.1e-6** Pa for A and B, respectively
- In TMAP7, the partial pressures are 1e-6 Pa and **5e-7** Pa for A and B, respectively

Here, it seems like we are reproducing TMAP7's case instead of TMAP4's. They are the same for a verification standpoint, but we want to be fully transparent. 
Could you please make a note of that in the documentation, the inputs, and in this python script and reproduce all three cases: Equal, different with TMAP4 concentrations, and different with TMAP7 concentrations?